### ISSUE TITLE
[graphicsmagick] build failure

### ISSUE THREAD

**Issue by hahamengbier** [microsoft/vcpkg#27988]
Package: graphicsmagick[core]:x86-windows -> 1.3.37#1

**Host Environment**

- Host: x64-windows
- Compiler: MSVC 19.34.31933.0
- vcpkg-tool version: 2022-11-10-5fdee72bc1fceca198fb1ab7589837206a8b81ba
  vcpkg-scripts version: c2c25d916 2022-11-23 (6 hours ago)

**To Reproduce**

`vcpkg install graphicsmagick`

**Failure logs**

```
-- Using cached GraphicsMagick-1.3.37-windows.7z.
-- Cleaning sources at C:/dev/vcpkg/buildtrees/graphicsmagick/src/2e465a2909-adf106de54.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source C:/dev/vcpkg/downloads/GraphicsMagick-1.3.37-windows.7z
-- Applying patch dynamic_bzip2.patch
-- Applying patch disable_graphicsmagick_modules.patch
-- Using source at C:/dev/vcpkg/buildtrees/graphicsmagick/src/2e465a2909-adf106de54.clean
-- Found external ninja('1.11.0').
-- Configuring x86-windows
-- Building x86-windows-dbg
-- Building x86-windows-rel
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:131 (message):
    Command failed: "C:/Program Files/CMake/bin/cmake.exe" --build . --config Release --target install -- -v -j13
    Working Directory: C:/dev/vcpkg/buildtrees/graphicsmagick/x86-windows-rel
    See logs for more information:
      C:\dev\vcpkg\buildtrees\graphicsmagick\install-x86-windows-rel-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_cmake.cmake:78 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_install_cmake.cmake:18 (vcpkg_build_cmake)
  ports/graphicsmagick/portfile.cmake:30 (vcpkg_install_cmake)
  scripts/ports.cmake:147 (include)



```

<details><summary>C:\dev\vcpkg\buildtrees\graphicsmagick\install-x86-windows-rel-out.log</summary>

```
[1/209] C:\PROGRA~1\MIB055~1\2022\COMMUN~1\VC\Tools\MSVC\1434~1.319\bin\Hostx64\x86\cl.exe   -DDLL -DMAGICK_IMPLEMENTATION -DWIN32 -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -D_DLL -D_MAGICKLIB_ -D_VISUALC_ -D_WANDLIB_ -D_WIN32 -D_WIN32_WINNT=0x0501 -Dgraphicsmagick_EXPORTS -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\. -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\magick -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++ -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\wand -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\jbig\libjbig -external:IC:\dev\vcpkg\installed\x86-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /MP  /MD /O2 /Oi /Gy /DNDEBUG /Z7 /showIncludes /FoCMakeFiles\graphicsmagick.dir\coders\caption.c.obj /FdCMakeFiles\graphicsmagick.dir\ /FS -c C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\coders\caption.c
Magick lib DLL export interface
Magick module DLL export interface
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\magick/nt_base.h(95): warning C4005: “HAVE_FT2BUILD_H”: 宏重定义
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\magick/magick_config.h(123): note: 参见“HAVE_FT2BUILD_H”的前一个定义
[2/209] C:\PROGRA~1\MIB055~1\2022\COMMUN~1\VC\Tools\MSVC\1434~1.319\bin\Hostx64\x86\cl.exe   -DDLL -DMAGICK_IMPLEMENTATION -DWIN32 -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -D_DLL -D_MAGICKLIB_ -D_VISUALC_ -D_WANDLIB_ -D_WIN32 -D_WIN32_WINNT=0x0501 -Dgraphicsmagick_EXPORTS -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\. -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\magick -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++ -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\wand -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\jbig\libjbig -external:IC:\dev\vcpkg\installed\x86-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /MP  /MD /O2 /Oi /Gy /DNDEBUG /Z7 /showIncludes /FoCMakeFiles\graphicsmagick.dir\coders\braille.c.obj /FdCMakeFiles\graphicsmagick.dir\ /FS -c C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\coders\braille.c
Magick lib DLL export interface
Magick module DLL export interface
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\magick/nt_base.h(95): warning C4005: “HAVE_FT2BUILD_H”: 宏重定义
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\magick/magick_config.h(123): note: 参见“HAVE_FT2BUILD_H”的前一个定义
...
Skipped 1002 lines
...

C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\magick/cdl.h(1): warning C4828: 文件包含在偏移 0xad 处开始的字符，该字符在当前源字符集中无效(代码页 65001)。
[196/209] C:\PROGRA~1\MIB055~1\2022\COMMUN~1\VC\Tools\MSVC\1434~1.319\bin\Hostx64\x86\cl.exe   -DDLL -DMAGICK_IMPLEMENTATION -DWIN32 -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -D_DLL -D_MAGICKLIB_ -D_VISUALC_ -D_WANDLIB_ -D_WIN32 -D_WIN32_WINNT=0x0501 -Dgraphicsmagick_EXPORTS -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\. -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\magick -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++ -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\wand -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\jbig\libjbig -external:IC:\dev\vcpkg\installed\x86-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /MP  /MD /O2 /Oi /Gy /DNDEBUG /Z7 /showIncludes /FoCMakeFiles\graphicsmagick.dir\wand\magick_compat.c.obj /FdCMakeFiles\graphicsmagick.dir\ /FS -c C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\wand\magick_compat.c
Magick lib DLL export interface
Magick module DLL export interface
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\magick/nt_base.h(95): warning C4005: “HAVE_FT2BUILD_H”: 宏重定义
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\magick/magick_config.h(123): note: 参见“HAVE_FT2BUILD_H”的前一个定义
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\magick/magick_config.h(123): warning C4005: “HAVE_FT2BUILD_H”: 宏重定义
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\magick/nt_base.h(95): note: 参见“HAVE_FT2BUILD_H”的前一个定义
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\magick/cdl.h(1): warning C4828: 文件包含在偏移 0xad 处开始的字符，该字符在当前源字符集中无效(代码页 65001)。
[197/209] C:\PROGRA~1\MIB055~1\2022\COMMUN~1\VC\Tools\MSVC\1434~1.319\bin\Hostx64\x86\cl.exe   /TP -DDLL -DMAGICK_IMPLEMENTATION -DWIN32 -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -D_DLL -D_MAGICKLIB_ -D_VISUALC_ -D_WANDLIB_ -D_WIN32 -D_WIN32_WINNT=0x0501 -Dgraphicsmagick_EXPORTS -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\. -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\magick -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++ -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\wand -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\jbig\libjbig -external:IC:\dev\vcpkg\installed\x86-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /MD /O2 /Oi /Gy /DNDEBUG /Z7 /showIncludes /FoCMakeFiles\graphicsmagick.dir\Magick++\lib\Options.cpp.obj /FdCMakeFiles\graphicsmagick.dir\ /FS -c C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib\Options.cpp
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib\Options.cpp(10): warning C4005: “MAGICK_IMPLEMENTATION”: 宏重定义
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib\Options.cpp(10): note: 之前在命令行上声明的“MAGICK_IMPLEMENTATION”
Magick lib DLL export interface
Magick module DLL export interface
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\magick/cdl.h(1): warning C4828: 文件包含在偏移 0xad 处开始的字符，该字符在当前源字符集中无效(代码页 65001)。
Magick++ lib DLL export
[198/209] C:\PROGRA~1\MIB055~1\2022\COMMUN~1\VC\Tools\MSVC\1434~1.319\bin\Hostx64\x86\cl.exe   /TP -DDLL -DMAGICK_IMPLEMENTATION -DWIN32 -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -D_DLL -D_MAGICKLIB_ -D_VISUALC_ -D_WANDLIB_ -D_WIN32 -D_WIN32_WINNT=0x0501 -Dgraphicsmagick_EXPORTS -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\. -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\magick -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++ -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\wand -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\jbig\libjbig -external:IC:\dev\vcpkg\installed\x86-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /MD /O2 /Oi /Gy /DNDEBUG /Z7 /showIncludes /FoCMakeFiles\graphicsmagick.dir\Magick++\lib\ImageRef.cpp.obj /FdCMakeFiles\graphicsmagick.dir\ /FS -c C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib\ImageRef.cpp
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib\ImageRef.cpp(10): warning C4005: “MAGICK_IMPLEMENTATION”: 宏重定义
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib\ImageRef.cpp(10): note: 之前在命令行上声明的“MAGICK_IMPLEMENTATION”
Magick lib DLL export interface
Magick module DLL export interface
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\magick/cdl.h(1): warning C4828: 文件包含在偏移 0xad 处开始的字符，该字符在当前源字符集中无效(代码页 65001)。
Magick++ lib DLL export
[199/209] C:\PROGRA~1\MIB055~1\2022\COMMUN~1\VC\Tools\MSVC\1434~1.319\bin\Hostx64\x86\cl.exe   /TP -DDLL -DMAGICK_IMPLEMENTATION -DWIN32 -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -D_DLL -D_MAGICKLIB_ -D_VISUALC_ -D_WANDLIB_ -D_WIN32 -D_WIN32_WINNT=0x0501 -Dgraphicsmagick_EXPORTS -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\. -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\magick -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++ -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\wand -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\jbig\libjbig -external:IC:\dev\vcpkg\installed\x86-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /MD /O2 /Oi /Gy /DNDEBUG /Z7 /showIncludes /FoCMakeFiles\graphicsmagick.dir\Magick++\lib\Thread.cpp.obj /FdCMakeFiles\graphicsmagick.dir\ /FS -c C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib\Thread.cpp
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib\Thread.cpp(8): warning C4005: “MAGICK_IMPLEMENTATION”: 宏重定义
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib\Thread.cpp(8): note: 之前在命令行上声明的“MAGICK_IMPLEMENTATION”
Magick lib DLL export interface
Magick module DLL export interface
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\magick/cdl.h(1): warning C4828: 文件包含在偏移 0xad 处开始的字符，该字符在当前源字符集中无效(代码页 65001)。
Magick++ lib DLL export
[200/209] C:\PROGRA~1\MIB055~1\2022\COMMUN~1\VC\Tools\MSVC\1434~1.319\bin\Hostx64\x86\cl.exe   /TP -DDLL -DMAGICK_IMPLEMENTATION -DWIN32 -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -D_DLL -D_MAGICKLIB_ -D_VISUALC_ -D_WANDLIB_ -D_WIN32 -D_WIN32_WINNT=0x0501 -Dgraphicsmagick_EXPORTS -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\. -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\magick -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++ -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\wand -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\jbig\libjbig -external:IC:\dev\vcpkg\installed\x86-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /MD /O2 /Oi /Gy /DNDEBUG /Z7 /showIncludes /FoCMakeFiles\graphicsmagick.dir\Magick++\lib\Image.cpp.obj /FdCMakeFiles\graphicsmagick.dir\ /FS -c C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib\Image.cpp
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib\Image.cpp(8): warning C4005: “MAGICK_IMPLEMENTATION”: 宏重定义
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib\Image.cpp(8): note: 之前在命令行上声明的“MAGICK_IMPLEMENTATION”
Magick lib DLL export interface
Magick module DLL export interface
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\magick/cdl.h(1): warning C4828: 文件包含在偏移 0xad 处开始的字符，该字符在当前源字符集中无效(代码页 65001)。
Magick++ lib DLL export
[201/209] C:\PROGRA~1\MIB055~1\2022\COMMUN~1\VC\Tools\MSVC\1434~1.319\bin\Hostx64\x86\cl.exe   /TP -DDLL -DMAGICK_IMPLEMENTATION -DWIN32 -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -D_DLL -D_MAGICKLIB_ -D_VISUALC_ -D_WANDLIB_ -D_WIN32 -D_WIN32_WINNT=0x0501 -Dgraphicsmagick_EXPORTS -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\. -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\magick -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++ -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\wand -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\jbig\libjbig -external:IC:\dev\vcpkg\installed\x86-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /MD /O2 /Oi /Gy /DNDEBUG /Z7 /showIncludes /FoCMakeFiles\graphicsmagick.dir\Magick++\lib\STL.cpp.obj /FdCMakeFiles\graphicsmagick.dir\ /FS -c C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib\STL.cpp
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib\STL.cpp(8): warning C4005: “MAGICK_IMPLEMENTATION”: 宏重定义
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib\STL.cpp(8): note: 之前在命令行上声明的“MAGICK_IMPLEMENTATION”
Magick lib DLL export interface
Magick module DLL export interface
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\magick/cdl.h(1): warning C4828: 文件包含在偏移 0xad 处开始的字符，该字符在当前源字符集中无效(代码页 65001)。
Magick++ lib DLL export
[202/209] C:\PROGRA~1\MIB055~1\2022\COMMUN~1\VC\Tools\MSVC\1434~1.319\bin\Hostx64\x86\cl.exe   /TP -DDLL -DMAGICK_IMPLEMENTATION -DWIN32 -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -D_DLL -D_MAGICKLIB_ -D_VISUALC_ -D_WANDLIB_ -D_WIN32 -D_WIN32_WINNT=0x0501 -Dgraphicsmagick_EXPORTS -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\. -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\magick -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++ -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\wand -IC:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\jbig\libjbig -external:IC:\dev\vcpkg\installed\x86-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /MD /O2 /Oi /Gy /DNDEBUG /Z7 /showIncludes /FoCMakeFiles\graphicsmagick.dir\Magick++\lib\Pixels.cpp.obj /FdCMakeFiles\graphicsmagick.dir\ /FS -c C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib\Pixels.cpp
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib\Pixels.cpp(8): warning C4005: “MAGICK_IMPLEMENTATION”: 宏重定义
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\Magick++\lib\Pixels.cpp(8): note: 之前在命令行上声明的“MAGICK_IMPLEMENTATION”
Magick lib DLL export interface
Magick module DLL export interface
C:\dev\vcpkg\buildtrees\graphicsmagick\src\2e465a2909-adf106de54.clean\magick/cdl.h(1): warning C4828: 文件包含在偏移 0xad 处开始的字符，该字符在当前源字符集中无效(代码页 65001)。
Magick++ lib DLL export
ninja: build stopped: subcommand failed.
```

</details>

**Additional context**

**Comment by hahamengbier** [comment]
I think that this issue is a Visual Studio bug. Maybe we should wait for Visual Studio team fix this. ^_^

**Comment by LilyWangLL** [comment]
Because this is a Visual Studio bug, the ticket submitted on [Developer Community](https://developercommunity.visualstudio.com/report?space=62&entry=problem) can effectively increase the priority of this bug being fixed.
Closed as duplicate of #27170.
